sdcm/scylla_bench_thread.py

```diff
@@ -137,6 +137,10 @@ def __init__(self, stress_cmd, loader_set, timeout, node_list=None, round_robin=
         self.sb_mode: ScyllaBenchModes = ScyllaBenchModes(re.search(r"-mode=(.+?) ", stress_cmd).group(1))
         self.sb_workload: ScyllaBenchWorkloads = ScyllaBenchWorkloads(
             re.search(r"-workload=(.+?) ", stress_cmd).group(1))
+        # NOTE: scylla-bench doesn't have 'mixed' workload. Its hdr tag names are the same in all cases.
```

Comment:
do you have jobs that enable HDR it for s-b ?

Reply:
No, we don't. Also, scylla-bench code doesn't use HDR file logger as of now.
I wrote it here just because addition of a support for scylla-bench becomes relatively easy - the same as upcoming latte PR.